python_modules/libraries/dagster-k8s/dagster_k8s/executor.py

```diff
@@ -65,20 +66,37 @@ def k8s_job_executor(init_context: InitExecutorContext) -> Executor:
               env_config_maps: ...
               env_secrets: ...
               job_image: ... # leave out if using userDeployments
+
+    If used with K8sRunLauncher, values from it's config is used for omitted fields.
     """
 
     run_launcher = init_context.instance.run_launcher
     exc_cfg = init_context.executor_config
+
+    _launcher_run_config = {
+        key: getattr(run_launcher, key, None)
+        for key in DagsterK8sJobConfig.config_type_pipeline_run().keys()
+    }
```

Comment:
I'm not quite comfortable with this one, was not sure if there should be a check like the one in; https://github.com/dagster-io/dagster/blob/fc94a35176bf412c72f3fc90e1f6d75a4cb86aa1/python_modules/libraries/dagster-celery-k8s/dagster_celery_k8s/executor.py#L112-L116